vm/src/any/memory/generation.cpp

```diff
@@ -20,6 +20,12 @@ void generation::print()
 
 
 newGeneration::newGeneration(int32 &eden_size, int32 &surv_size, FILE *snap) {
+#if TARGET_OS_VERSION == CYGWIN_VERSION
+  int32 adjust_size = getpagesize();
+#else
+  int32 adjust_size = idealized_page_size;
+#endif
+
```

Comment:
Windows has two different page sizes, depending on what you're doing. I was hoping to avoid using the larger size for everything, but that is starting to go against the grain too much, so I will probably end up just using the larger size.